com.unity.multiplayer.mlapi/Tests/Runtime/Components/NetworkUpdateStagesComponent.cs

```diff
@@ -32,14 +32,14 @@ public class NetworkUpdateStagesComponent : NetworkBehaviour
         // Start is called before the first frame update
         private void Start()
         {
-            m_ServerParams.Send.UpdateStage = NetworkUpdateStage.Initialization;
+            m_ServerParams.Send.UpdateStage = NetworkUpdateStage.EarlyUpdate;
             m_ClientParams.Send.UpdateStage = NetworkUpdateStage.Update;
```

Comment:
Should this also be possibly set to `EarlyUpdate` (idk if it's logically so - I'm just highlighting that all the other variables have matching changes)

Reply:
I just changed the ones that were set to Initialization since Initialization isn't a valid update stage for RPCs.